aws_lambda_powertools/utilities/idempotency/idempotency.py

```diff
@@ -151,7 +151,7 @@ def _get_idempotency_record(self) -> DataRecord:
 
         """
         try:
-            event_record = self.persistence_store.get_record(self.event)
+            event_record = self.persistence_store.get_record(self.event, self.context)
```

Comment:
```suggestion
            event_record = self.persistence_store.get_record(event=self.event, context=self.context)
```

Makes sense to remain consistent